ucm2/blobs/sof/ipc4/drc/README.md

```diff
@@ -1,7 +1,7 @@
 # How to build
 
-These blobs were exported with example_drc.m tool from
```

Comment:
@perexg @lgirdwood I will change the file suffix from .blob to .bin for added blob. Also I'll change the existing ones to .bin to have similar looking file names.

Reply:
Done now.

Reply:
Explanation https://github.com/thesofproject/sof/pull/9711#discussion_r1876206310